### DISCUSSION TITLE
Can we style the iOS blue system color for ImagePicker?

### DISCUSSION THREAD

**Discussion by newyorksoul** [expo/expo#18244] [Archived: Questions]
![image](https://user-images.githubusercontent.com/45617091/179089020-6b576c75-9092-4bf3-93af-bd8b4f90ba65.png)

**Comment by barthap** [comment]
Any more details about this? This screenshot doesn't tell me much. Do you know if there is any setting/property for this look in native iOS [`PHPickerViewController` ](https://developer.apple.com/documentation/photokit/phpickerviewcontroller)/ [`UIImagePickerController`](https://developer.apple.com/documentation/uikit/uiimagepickercontroller)?
